lnrpc/rpc.proto

```diff
@@ -1442,6 +1442,9 @@ message Invoice {
     invoice's destination.
     */
     repeated RouteHint route_hints = 14 [json_name = "route_hints"];
+
+    /// Whether this invoice should include routing hints for private channels.
+    bool private = 15 [json_name = "private"];
```

Comment:
Perhaps we should just make this the default on the `lncli` level? Nice that users are able to toggle it in any case.